test/test_export.py

```diff
@@ -49,3 +49,19 @@ def test_export(
 
     # Remove the temporary file
     os.system(f"rm ./pytximport_cli_test_{current_time}.h5ad")  # nosec
+
+    # Test saving as summarizedexperiment
+    _ = tximport(
+        salmon_multiple_files,
+        "salmon",
+        transcript_gene_mapping_path_mouse,
+        output_type="summarizedexperiment",
+        output_format="summarizedexperiment",
+        output_path=f"./pytximport_cli_test_{current_time}",
+    )
+
+    # Check that the output file was created
+    assert os.path.exists(f"./pytximport_cli_test_{current_time}"), "SummarizedExperiment output file was not created."
+
+    # Remove the temporary file
+    os.system(f"rm ./pytximport_cli_test_{current_time}")  # nosec
```

Comment:
File type extension?

Reply:
the language agnostic representations are stored in a directory following a structure similar to Bioconductor representations. There's no file format extension to these directories.